src/API.js

```diff
@@ -0,0 +1,36 @@
+export default (() => {
+  const sendScores = async (url, userName, userScore) => {
+    const gameResult = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+      body: JSON.stringify({
+        user: userName.value,
+        score: userScore.value,
+      }),
+    });
+    userName.value = '';
+    userScore.value = '';
+    return gameResult.json();
+  };
```

Comment:
- [ ] Scores have to be a positive integer (currently I can add strings such as `Ramo`). please add some validation to your input field so it accepts only numbers.